torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1519,6 +1519,56 @@ at::Tensor leaky_relu_backward(const at::Tensor& grad_output,
       negative_slope.to<double>()));
 }
 
+at::Tensor lerp(const at::Tensor& self, const at::Tensor& end,
+                const at::Tensor& weight) {
+  XLA_FN_COUNTER("xla::");
+  return bridge::AtenFromXlaTensor(
+      XLATensor::lerp(bridge::GetXlaTensor(self), bridge::GetXlaTensor(end),
+                      bridge::GetXlaTensor(weight)));
+}
+
+at::Tensor lerp(const at::Tensor& self, const at::Tensor& end,
+                const at::Scalar& weight) {
+  XLA_FN_COUNTER("xla::");
+  return bridge::AtenFromXlaTensor(XLATensor::lerp(
+      bridge::GetXlaTensor(self), bridge::GetXlaTensor(end), weight));
+}
+
+at::Tensor& lerp_(at::Tensor& self, const at::Tensor& end,
+                  const at::Tensor& weight) {
+  XLA_FN_COUNTER("xla::");
+  XLATensor self_tensor = bridge::GetXlaTensor(self);
+  XLATensor::lerp_(self_tensor, bridge::GetXlaTensor(end),
+                   bridge::GetXlaTensor(weight));
+  return self;
+}
+
+at::Tensor& lerp_(at::Tensor& self, const at::Tensor& end,
+                  const at::Scalar& weight) {
```

Comment:
cc: @bdhirsh , do we still expect to write out inplace version in the future? ;)

Reply:
@JackCaoG you can just implement `lerp.Scalar` and `lerp.Tensor`, and the other 4 versions of those ops (both inplace and functional) will be generated for you :). We probably still want the tests though.

I don't think we can expect to **never** write out=/inplace kernels though; we can only do it for operators that have a functional and out= overload in native_functions.yaml (so e.g. that doesn't apply to `random_`, which XLA still implements)

Reply:
Ah, I forgot about that change! This is pretty convenient 😆